.gitignore

```diff
@@ -6,12 +6,16 @@ __pycache__
 Pipfile.lock
 build/
 *.egg-info/
-
-# mypy
-.mypy_cache/
+dist/
 
 # IDE settings
 .vscode/
 .idea/
 .idea/taipy.iml
 .DS_Store
+
+# Core .data directory
+.data/
+
+# demo files
+demo-*
```

Comment:
It's surprising to me to ignore all files starting with `demo-`. Is it a general rule you have on the GUI repo?

Reply:
yes